src/process.cr

```diff
@@ -414,6 +414,27 @@ class Process
   private def close_io(io)
     io.close if io
   end
+
+  # Changes the root directory and the current working directory for the current
+  # process.
```

Comment:
It should probably be obvious to everyone using it, but it wouldn't hurt to add a note that chroot should not be used as a security mechanism (especially if the process does not drop privilege).

Reply:
I have addressed this in 7361ad6.

Reply:
Didn't need to be so detailed ;) A simple "Chroot should not be used as security measure." would have been fine.

Reply:
Maybe add that as a first sentence to make it Crystal clear? 😄

Reply:
I don't think "chroot should not be used as security measure." is correct. It can be used effectively as a mitigation, though its not perfect.